dvc/remote/ssh/__init__.py

```diff
@@ -122,6 +123,18 @@ def remove(self, path_info):
         with self.ssh(**path_info) as ssh:
             ssh.remove(path_info["path"])
 
+    def cache_exists(self, checksums):
+        # Check each checksum if we have less then defined num of checksums
+        if len(checksums) < self.CHECK_EXIST_LIST_ALL_FILE_THRESHOLD:
+            with self.ssh(**self.path_info) as ssh:
```

Comment:
How about extracting this to something like _check_each_checksum? Then the comment is unnecessary.